pom.xml

```diff
@@ -537,6 +537,10 @@
                     <exclude>commons-logging:commons-logging:*:jar:runtime</exclude>
                   </excludes>
                 </bannedDependencies>
+                <requireReleaseDeps>
+                  <message>No Snapshots Allowed For Release Versions</message>
+                  <onlyWhenRelease>true</onlyWhenRelease>
+                </requireReleaseDeps>
```

Comment:
For some corner cases not caught by `maven-release-plugin` when there's a `release.properties` from a previous build.